src/datta_lab_to_nwb/markowitz_gillis_nature_2023/convert_session.py

```diff
@@ -51,7 +51,7 @@ def session_to_nwb(
     conversion_options.update(dict(FiberPhotometry=dict()))
     conversion_options.update(dict(Behavior=dict()))
 
-    converter = MarkowitzGillisNature2023NWBConverter(source_data=source_data)
+    converter = markowitz_gillis_nature_2023.NWBConverter(source_data=source_data)
```

Comment:
It's an interesting style to be sure

We technically support this style for `tools` submodules in NeuroConv (you can do direct `from neuroconv.tools.spikeinterface import write_recording` or `from neuroconv.tools import spikeinterface; spikeinterface.write_recording(...)`)

First time seeing it used for a conversion project, though... could be interesting

Only problem I foresee if for bigger projects it could conflate abstraction (when you start mixing common top-level interfaces in, like a submodule `datta_lab_to_nwb.common_interfaces` then `common_interfaces.SomeTypeOfCommonBehaviorInterface` could get mixed with `specific_sub_project.SpecificBehaviorInterface`, which reads oddly)